### ISSUE TITLE
creating talk page using rmd instead of md

### ISSUE THREAD

**Issue by jananiravi** [sourcethemes/project-kickstart-r#3]
The following yaml works perfectly well while listing all talks and the talk page using the slug, but doesn't appear on the home page under #talks widget:

```
---
title: R-Ladies East Lansing | Workshop on Intro to Tidy Data w/ R
author: Janani Ravi
date: '2018-11-05'
slug: rlel-workshop-tidydata
categories:
  - R-Ladies
  - Resources
tags:
  - R
  - rladies
  - datascience
  - software-tools
image:
  caption: ''
  focal_point: ''
selected: true
time_start: 2018-11-05T18:00:00
time_end: 2018-11-05T20:00:00
abstract: ""
abstract_short: ""
event: "R-Ladies East Lansing | November 5, 2018"
event_url: "https://rladies-eastlansing.github.io"
location: "East Lansing, MI"
---
```

Am I missing something here, @gcushen 
Also, to include images inline, should I create a separate folder using the format `X/index.rmd` I understand this works for featured images. How about inserting other images inline for talks/posts? Inserting images from /static/img doesn't seem to be working. thanks!

**Comment by gcushen** [comment]
The content in this repo is currently for Academic v2.4.0. To use features of the latest version of Academic requires [migrating the content front matter and configuration](https://sourcethemes.com/academic/docs/update/). For v3+, the featured image documentation can be found here: https://sourcethemes.com/academic/docs/managing-content/#featured-image

**Comment by jananiravi** [comment]
Thank you, @gcushen! Is there an example RMarkdown `yaml` chunk for talk/other templates for `v3+`?

**Comment by gcushen** [comment]
No, but there are numerous online TOML-YAML convertors and guides on Google.

**Comment by jananiravi** [comment]
True, @gcushen that's how I got to the `rmd yaml` chunk posted above. Doesn't seem to be working as well as the `.md`, though. For instance, the above chunk doesn't help the post show up in the list of talks on the home page. @apreshill do you happen to have working conversions for the latest academic v3.1 md -> rmd? thanks!

**Comment by apreshill** [comment]
no sorry-I'm no help here!

**Comment by gcushen** [comment]
Bear in mind that if something works in Markdown but not RMarkdown, then it is most likely an issue with RStudio/Blogdown and I suggest raising the issue with that project.

RStudio/Blogdown is a wrapper around Academic and Hugo - Academic is not involved at all in the the RMarkdown processing.

**Comment by gcushen** [comment]
@jananiravi I have just tested your talk front matter by putting it in a `.Rmarkdown` file and adding the talk widget from Academic's example site to the `content/home/` folder. Everything is working fine (it displays as expected in the talk widget and on it's own page) with the latest versions of Project KickstartR, Academic, Hugo, and Blogdown v0.8.

Note that Project KickstartR (such as the config and front matter of files) was updated earlier today to be compatible with Academic v3.1.
